src/utils.js

```diff
@@ -127,7 +139,28 @@ const api = {
 	setGitConfig(name, value) {
 		return api.exec(`${GIT_CONFIG_COMMAND} ${name} ${value.trim()}`);
```

Comment:
I'd like the git username/token put in an RC file instead of git config. The idea is that in the next release we can stop reading from git config altogether, but to do that we need to take over that storage in our RC file. 

The easily solution would probably be for `getGitConfig` to see if those values are present in git config and if they are, delete them, and put them in the ROOT RC file instead (update or create from scratch).

`setGitConfigs` (probably rename to something else) would update or create from scratch an RC file at the ROOT with those username/token values.